src/tss2-fapi/ifapi_keystore.c

```diff
@@ -644,6 +644,7 @@ ifapi_keystore_load_finish(
     if (jso)
         json_object_put(jso);
     LOG_TRACE("Return %x", r);
+    object->rel_path = NULL;
     SAFE_FREE(keystore->rel_path);
```

Comment:
This is dead code now, so we should remove it.

Reply:
LGTM on line 632 object->_rel_path is assigned the value of keystore->rel_path (pointer) for the purpose of calling ifapi_json_IFAPI_OBJECT_deserialize. Which if that fails we free the keystore->rel_path and leave object->_rel_path dangling which is then freed later in ifapi_cleanup_ifapi_object.

Reply:
> > This is dead code now, so we should remove it.
> 
> @AndreasFuchsTPM Why is there dead code?

my mistake. I misread the code